src/preprocess.py

```diff
@@ -60,7 +60,11 @@ def preprocess(file):
     y = label
         The label is the 'defects' column in dataset table
     """
-    X = pd.DataFrame(dataset.drop(["defects"], axis=1))
+    
+    # Select only feature with the absolute of correlation > 0.1
+    features = dataset.corr()["defects"].loc[abs(dataset.corr()["defects"]) > 0.1].sort_values().index.values.tolist()
+    
+    X = pd.DataFrame(dataset[features[:-1])
```

Comment:
kurang satu kurung ']'
```suggestion
    X = pd.DataFrame(dataset[features[:-1]])
```

Reply:
oiya lupa